### ISSUE TITLE
`Exceeded limit on max bytes to buffer` error when invoking export API

### ISSUE THREAD

**Issue by blindpirate** [gradle/bt-dev-prod-data-collector#30]
```
Caused by: org.springframework.core.io.buffer.DataBufferLimitException: Exceeded limit on max bytes to buffer : 524288
at org.springframework.core.io.buffer.LimitedDataBufferList.raiseLimitException(LimitedDataBufferList.java:99)
Suppressed: The stacktrace has been enhanced by Reactor, refer to additional information below:
Error has been observed at the following site(s):
 *__checkpoint ⇢ Body from GET https://ge.gradle.org/build-export/v2/build/ealhxkl3qmjks/events?eventTypes=BuildStarted,BuildFinished,BuildCacheRemoteLoadFinished,BuildCacheRemoteStoreFinished,TestStarted,TestFinished,TaskStarted,UserTag,UserNamedValue,ProjectStructure,BuildAgent,DaemonState,DaemonUnhealthy,TaskFinished [DefaultClientResponse]
```

**Comment by blindpirate** [comment]
See https://stackoverflow.com/questions/59735951/databufferlimitexception-exceeded-limit-on-max-bytes-to-buffer-webflux-error

**Comment by blindpirate** [comment]
Fixed in #31
